release/src/main/scripts/build_release_candidate.sh

```diff
@@ -102,31 +104,12 @@ if [[ $confirmation = "y" ]]; then
   fi
   mkdir -p ${LOCAL_CLONE_DIR}
   cd ${LOCAL_CLONE_DIR}
-  git clone ${GIT_REPO_URL}
+  git clone --depth 1 --branch "${RC_TAG}" ${GIT_REPO_URL} "${BEAM_ROOT_DIR}"
   cd ${BEAM_ROOT_DIR}
-  git checkout ${RELEASE_BRANCH}
-  RELEASE_COMMIT=$(git rev-parse --verify ${RELEASE_BRANCH})
 
   echo "-------------Building Java Artifacts with Gradle-------------"
   git config credential.helper store
 
-  if git rev-parse "v${RELEASE}-RC${RC_NUM}" >/dev/null 2>&1; then
```

Comment:
Is this case (tag already exists) still handled ?

Reply:
Let me test. I think that it will already just mv the tag.

Reply:
It turns out no. I will need to update the choose_rc_commit.sh to have an `--overwrite` option, or the user will just have to issue the command. It is a git one liner so it doesn't help very much. It should be pretty rare now that choosing a commit is totally separate from building the RC.

1. You find a green commit and choose it.
2. You can try to build the RC as many times as you need.

Reply:
Done